tendermint/src/public_key.rs

```diff
@@ -91,6 +91,13 @@ impl PublicKey {
     }
 }
 
+impl Default for PublicKey {
+    fn default() -> PublicKey {
+        static DUMMY_BYTES: [u8; 32] = [0; 32];
+        PublicKey::Ed25519(ed25519::PublicKey::new(DUMMY_BYTES))
```

Comment:
This also doesn’t make any sense, and is a particularly pathological case as it’s a special point on the elliptic curve.